force-app/main/default/flows/Enter_Journey_Details.flow-meta.xml

```diff
@@ -0,0 +1,639 @@
+            <operator>Assign</operator>
+            <value>
+                <elementReference>Unchanged_Task_Loop.Task_Description__c</elementReference>
+            </value>
+        </assignmentItems>
+        <assignmentItems>
+            <assignToReference>varCurrentUneditedTemplateTask.Task_Relative_Days_To_Complete__c</assignToReference>
+            <operator>Assign</operator>
+            <value>
+                <elementReference>Unchanged_Task_Loop.Task_Relative_Days_To_Complete__c</elementReference>
+            </value>
+        </assignmentItems>
+        <assignmentItems>
+            <assignToReference>varCurrentUneditedTemplateTask.Template__c</assignToReference>
+            <operator>Assign</operator>
+            <value>
+                <elementReference>Unchanged_Task_Loop.Template__c</elementReference>
+            </value>
+        </assignmentItems>
+        <connector>
+            <targetReference>Assign_Unedited_Details_To_Journey_Task</targetReference>
+        </connector>
+    </assignments>
+    <assignments>
+        <description>Assigns the user&apos;s edits in the screen flow to a temporary Template Task variable to store the details for each task in the collection.</description>
+        <name>Capture_Task_Edits</name>
+        <label>Capture Task Edits</label>
+        <locationX>138</locationX>
+        <locationY>890</locationY>
+        <assignmentItems>
+            <assignToReference>varCurrentTemplateTask.Name</assignToReference>
+            <operator>Assign</operator>
+            <value>
+                <elementReference>Task_Name</elementReference>
+            </value>
+        </assignmentItems>
+        <assignmentItems>
+            <assignToReference>varCurrentTemplateTask.Task_Description__c</assignToReference>
+            <operator>Assign</operator>
+            <value>
+                <elementReference>Task_Description</elementReference>
+            </value>
+        </assignmentItems>
+        <assignmentItems>
+            <assignToReference>varCurrentTemplateTask.Task_Relative_Days_To_Complete__c</assignToReference>
+            <operator>Assign</operator>
+            <value>
+                <elementReference>Relative_Days_to_Complete</elementReference>
+            </value>
+        </assignmentItems>
+        <assignmentItems>
+            <assignToReference>varCurrentTemplateTask.Template__c</assignToReference>
+            <operator>Assign</operator>
+            <value>
+                <elementReference>Task_Loop.Template__c</elementReference>
+            </value>
+        </assignmentItems>
+        <connector>
+            <targetReference>Assign_Task_Details_To_Journey_Task</targetReference>
+        </connector>
+    </assignments>
+    <decisions>
+        <description>Decision element with 2 paths depending on whether the user checked the &quot;edit template tasks&quot; checkbox.</description>
+        <name>Edit_Tasks</name>
```

Comment:
This works as expected, **IF** the Journey you're creating from a template has tasks. It's not wrong but should I have the option to edit tasks if they don't exist? Wouldn't I just press create?